src/main/java/gift/exception/GlobalExceptionHandler.java

```diff
@@ -39,6 +39,42 @@ public ResponseEntity<ExceptionResponseDto> handleIllegalArgumentException(
         return ResponseEntity.badRequest().body(response);
     }
 
+    @ResponseBody
+    @ExceptionHandler(EmailDuplicationException.class)
+    public ResponseEntity<ExceptionResponseDto> handleEmailDuplicationException(
+        EmailDuplicationException e) {
+        List<String> errors = new ArrayList<>();
+        errors.add(e.getMessage());
+        ExceptionResponseDto exception = new ExceptionResponseDto(errors, 409,
```

Comment:
status code를 직접 하드코딩하기보다는 가급적 이미 구현되어있는 상수를 활용해주는 것이 실수할 여지를 없앨 수 있는 것 같아요. 😃

```suggestion
        ExceptionResponseDto exception = new ExceptionResponseDto(errors, HttpStatus.CONFLICT.value(),
```

Reply:
HttpStatus를 활용하는 방식으로 변경했습니다